boilerPlate/e2e/protractor.conf.js

```diff
@@ -0,0 +1,37 @@
+// @ts-check
```

Comment:
Remove all testing related files as we are not using.

Reply:
Done